packages/legend-graph/src/graphManager/protocol/pure/v1/transformation/pureGraph/to/V1_ElementSecondPassBuilder.ts

```diff
@@ -251,6 +251,17 @@ export class V1_ElementSecondPassBuilder
       V1_buildVariable(param, this.context),
     );
     func.expressionSequence = protocol.body;
+    // Make sure func.name consists of function name plus function signature
```

Comment:
why are we doing this the second pass. 
Wouldnt it be better to just ensure `func`'s name is the full name during the FirstPass when we set the name? Most likely prevent us from having to call renameOwnelement during the builders.

Reply:
This logic is also used for existing functions of which file name only contains function name (shorter version) in Gitlab, and it can't be done during the firstPass because parameter, returnType/ multiplicity are processed during the secondPass.

Reply:
moved to firstPass